homeassistant/components/apple_tv.py

```diff
@@ -91,7 +91,7 @@ def configuration_callback(callback_data):
         hass.async_add_job(configurator.request_done, instance)
 
     instance = configurator.request_config(
-        hass, 'Apple TV Authentication', configuration_callback,
+        'Apple TV Authentication', configuration_callback,
```

Comment:
The point release would _only_ include the fixes for Axis + AppleTV, not for Automatic. We never ship breaking changes in point releases.

(the configurator change has already been released as part of 0.51.2 because I had to do a frontend update and didn't see that change as a big enough reason to leave it for 0.52)